data/store/mongo/mongo_data.go

```diff
@@ -949,43 +954,67 @@ func (d *DataRepository) GetLastUpdatedForUser(ctx context.Context, id string) (
 		return nil, errors.New("id is missing")
 	}
 
-	var dataSet []*continuous.Continuous
+	futureCutoff := time.Now().AddDate(0, 0, 1).UTC()
+	pastCutoff := time.Now().AddDate(-2, 0, 0).UTC()
+
+	selectorOld := bson.M{
+		"_active": true,
+		"_userId": id,
+		"type":    "cbg",
+		"time": bson.M{"$lte": futureCutoff.Format(time.RFC3339Nano),
+			"$gte": pastCutoff.Format(time.RFC3339Nano)},
+	}
+
 	selector := bson.M{
 		"_active": true,
 		"_userId": id,
 		"type":    "cbg",
+		"time": bson.M{"$lte": futureCutoff,
+			"$gte": pastCutoff.Format(time.RFC3339Nano)},
```

Comment:
Shouldn't both of these be dates (i.e. not strings)?

Reply:
good catch, this would be a weird bug to track down after date migration.